config/cypress/cypress.config.js

```diff
@@ -32,11 +32,16 @@ const resourceTypes = /\.(png|svg|eot|woff|woff2|ttf|otf|svg(#Symbola)?)$/;
 module.exports = defineConfig({
     fixturesFolder: false,
     video: false,
-
+    // Prevent Cypress from scrolling to elements before clicking them.
+    scrollBehavior: false,
```

Comment:
This prevents Cypress from auto-scrolling to the node we're clicking... which was messing up our coordinate calculations of where we wanted to drag to!